dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/trackedentity/DefaultTrackerAccessManager.java

```diff
@@ -62,7 +62,7 @@
 public class DefaultTrackerAccessManager implements TrackerAccessManager {
 
   private final AclService aclService;
-  private final TrackerOwnershipManager ownershipAccessManager;
+  private final ApiTrackerOwnershipManager ownershipAccessManager;
```

Comment:
As this is the only place outside tracker where we are using `TrackerOwnershipManager`, wouldn't be easier to move this class in this same PR and we avoid the need to create  `ApiTrackerOwnershipManager`

Reply:
I can't move this class into tracker because it's used by the `DefaultTrackedEntityChangeLogService`, which is also outside our module, and it's in turn used by `DefaultProgramMessageService` and `DeliveryChannelStrategy`, which are also outside. This is one of the things I'd like discuss.